app/javascript/packages/document-capture/interface/documents-image-selfie-value.tsx

```diff
@@ -4,6 +4,7 @@ export type ImageValue = Blob | string | null | undefined;
 export interface DocumentsAndSelfieStepValue {
   front: ImageValue;
   back: ImageValue;
+  passport: ImageValue;
   selfie: ImageValue;
   front_image_metadata?: string;
```

Comment:
question (non-blocking): Is no metadata captured for the passport image or are we storing it in the `front_image_metadata` field?

Reply:
Good catch! Done in e7ae83bcd7